testing/tests/DevExpress.ui.widgets.editors/numberBoxParts/common.tests.js

```diff
@@ -2019,7 +2021,19 @@ QUnit.module('number validation', {}, () => {
 
         const instance = $numberBox.dxNumberBox('instance');
 
-        assert.equal($numberBox.find('.dx-invalid-message .dx-overlay-content').text(), instance.option('invalidValueMessage'), 'validation message is rendered');
+        assert.equal($numberBox.find(INVALID_MESSAGE_POPUP_CONTENT_SELECTOR).text(), instance.option('invalidValueMessage'), 'validation message is rendered');
```

Comment:
Shoudn't we use `strictEqual`?

Reply:
I think we should fix all minor test problems out of the context of this PR